staging/src/k8s.io/kubectl/pkg/cmd/run/run.go

```diff
@@ -173,6 +173,7 @@ func NewCmdRun(f cmdutil.Factory, streams genericclioptions.IOStreams) *cobra.Co
 
 func addRunFlags(cmd *cobra.Command, opt *RunOptions) {
 	cmdutil.AddDryRunFlag(cmd)
+	cmd.Flags().StringArray("annotation", []string{}, "Annotations to apply to the pod.")
```

Comment:
Wrap the description in `i18n.T()` like the others. This is used for internationalization/translation.

Reply:
Right, my bad. Shall i also add `i18n.T()` to labels in this PR or only wrap it around annotations?